product-updates/security-manager-2-release-note.md

```diff
@@ -8,13 +8,31 @@ SysKit Security Manager is centralized __SharePoint and Office 365 security repo
 [Try SysKit Security Manager Free](https://www.syskit.com/products/security-manager/download/)
 
 __Product version:__ 2.0.0  
-__Build number:__ 590      
+__Build number:__ 599     
 __Release date:__ May 16, 2018  
 
 ## Features
-* __SharePoint Online and On-Premises Snapshots__ are now available in SysKit Security Manager! In addition to being able to manage SharePoint Online and On-Premises permissions, you can now take snapshots of all your site collections and Office 365 groups.
+* __SharePoint Online and On-Premises Snapshots__ are now available in SysKit Security Manager! In addition to being able to manage SharePoint Online and On-Premises permissions, you can now take snapshots of all your site collections and Office 365 groups. This is required in order to use more advanced __Permission Reports__
```

Comment:
...to use _the_ more advanced..,?

Reply:
i really don't know...

Reply:
točka na kraj? I još ne znam jel permissions reports ili permission reports. imamo oboje u aplikaciji.

Reply:
I think without is the more correct version, but I'm never sure about details like this.